xtask/src/release.rs

```diff
@@ -99,7 +99,7 @@ impl PromoteCmd {
         cmd!("git add src/tools/rust-analyzer").run()?;
         cmd!("git commit -m':arrow_up: rust-analyzer'").run()?;
         if !self.dry_run {
-            cmd!("git push").run()?;
+            cmd!("git push origin {branch}").run()?;
```

Comment:
```suggestion
            cmd!("git push -u").run()?;
```

I think this should work?

Reply:
I'm not sure, we'll see next week, I guess?

Reply:
It doesn't.